### ISSUE TITLE
Vurdere lisenstype

### ISSUE THREAD

**Issue by mikaelbr** [varianter/dokument-mal#8]
Er MIT lisens det beste for dokumenter på denne måten? Ønsker vi å sette noen retning for hvordan de brukes? Hva med noe radikalt som public domain eller unlicense? Her kan vi gjøre en vurdering.

**Comment by oddsve** [comment]
jeg er litt usikker på de ulike open source lisensene må jeg innrømme. 
Det jeg tenker er viktig er at aldri må få ansvar for at noen bruker det?

**Comment by mikaelbr** [comment]
Ja, det tenker jeg og. Derfor jeg defaulta til MIT (se oppsummering på toppen: https://github.com/varianter/dokument-mal/blob/master/LICENSE.md). Men det er mange andre som har det og. Jeg er ingen ekspert selv, men tenker så lenge vi ikke har noen sterke krav så holder MIT fint. Det er kompatibelt med andre lisenser og eneste føringen den legger er at lisensen må være inkludert i systemer som bruker det.
